src/components/atoms/Avatar/index.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { render } from "@testing-library/react";
+import Avatar from ".";
+import AvatarSvg from "../../../../public/assets/avatar.svg";
+
+describe('Avatar tests', () => {
+    it('renders Avatar with default props and should match the snapshot', () => {
+        const { container } = render(<Avatar isImage={false} />);
+        expect(container.firstChild).toMatchSnapshot();
```

Comment:
instead of .toMatchSnapshot you can follow methods like toBeInTheDocument, getBy, findBy, queryBy, 

https://www.youtube.com/watch?v=7dTTFW7yACQ&list=PL4cUxeGkcC9gm4_-5UsNmLqMosM-dzuvQ
Crash course of react testing library.